android/tangram/src/com/mapzen/tangram/Marker.java

```diff
@@ -0,0 +1,179 @@
+package com.mapzen.tangram;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Drawable;
+import android.util.DisplayMetrics;
+
+import com.mapzen.tangram.geometry.Polygon;
+import com.mapzen.tangram.geometry.Polyline;
+
+/**
+ * Class used to display points, polylines, and bitmaps dynamically on a map. Do not create one of
+ * these objects directly, instead use {@link MapController#addMarker()}.
+ */
+public class Marker {
+
+    Context context;
+    long markerId = 0;
```

Comment:
How does `FeaturePickListener` work with the new marker API? Is the `markerId` returned as a property? Do we need a public getter for `markerId` so we can match which feature is clicked in the callback?